.changeset/lazy-ways-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+dispatch custom event to retrieve composed path
```

Comment:
Maybe we can rephrase this as

```suggestion
fix `sveltekit:prefetch` mouse detection
```

So that it's identifiable as a prefetch fix in the changelog.